packages/@coorpacademy-components/src/molecule/setup-cohort-item/test/fixtures/default.js

```diff
@@ -0,0 +1,103 @@
+export default {
+  props: {
+    fields: [
+      {
+        type: 'text',
+        title: 'Name',
+        value: 'value',
+        required: true,
+        autoFocus: true
+      },
+      {
+        type: 'splitSreen',
+        conditions: {
+          title: 'The Provider',
+          values: [
+            {
+              type: 'checkbox',
+              title: 'Provider 1',
+              checked: true
```

Comment:
you will need the value field. Title is the label. it's the value will be saved.

For example :
```
{
  "value": "them_NyQ1kXpl7",
  "title":"Wellness at Work"
}
```
`Wellness at Work` is presented on the interface but `them_NyQ1kXpl7` is saved

Reply:
it's same for `poviders` and `groups`

Reply:
It's Done